stdlib/public/Concurrency/TaskSleepDuration.swift

```diff
@@ -142,4 +142,20 @@ extension Task where Success == Never, Failure == Never {
   ) async throws {
     try await clock.sleep(until: deadline, tolerance: tolerance)
   }
+  
+  /// Suspends the current task for the given duration on a continuous clock.
+  ///
+  /// If the task is canceled before the time ends, this function throws 
+  /// `CancellationError`.
+  ///
+  /// This function doesn't block the underlying thread.
+  ///
+  ///       try await Task.sleep(for: .seconds(3))
+  ///
```

Comment:
Let's fill in the missing parameter docs too:

```suggestion
  ///
  /// - Parameter duration: The duration to wait, in seconds.
```

Reply:
I don't think `in seconds` is correct here? `Duration` can be constructed from other units.